people/benefits.md

```diff
@@ -56,6 +56,10 @@ We encourage you to attend conferences to improve your knowledge of a subject ar
 
 All expenses for the conference, such as transport, accommodation, fees, food, etc., are covered by your [allowance](#productivity-allowance) and the additional conference allowance.
```

Comment:
Simplify the sentence to `All conference expenses such as transport, accommodation, fee...`